### PR TITLE
{bio}[foss/2021b] DRAGMAP v1.3.0

### PR DESCRIPTION
**Pull request by ItIsI-Orient**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ItIsI-Orient** [comment]
Test report by @ItIsI-Orient
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3702.slaking.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/39c9151b61050bd8038b5280ec9b69be for a full test report.

File: easybuild/easyconfigs/d/DRAGMAP/DRAGMAP-1.3.0-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+sources = ['%(version)s.tar.gz']
+checksums = ['baf546c2f53b13e2bce4c937201e38459f3ce2f1069722ca670d8726a4bae224']
+
+skipsteps = ['configure']
+prebuildopts = 'sed -i "s|-msse4.2 -mavx2|$CXXFLAGS|g" config.mk &&'
+buildopts = 'CC="$CC" CXX="$CXX"'
+preinstallopts = 'mkdir %(installdir)s/bin &&'
+installopts = 'DESTDIR=%(installdir)s'
+
+dependencies = {
```
Comment (smoors) [inline easybuild/easyconfigs/d/DRAGMAP/DRAGMAP-1.3.0-foss-2021b.eb:21]: @ItIsI-Orient for consistency with the other easyconfigs, please move the dependencies up to below the checksums?

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
node300.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/d8c08677291891d0dd6db1d1ebceacb9 for a full test report.

**Pull request merged**